scanner/scanner.go

```diff
@@ -344,54 +266,32 @@ func (s *Scanner) Scan(ctx context.Context, foundCert func(*ct.LogEntry), foundP
 		close(stop)
 	}()
 
-	fetches := make(chan fetchRange)
+	batches := make(chan EntryBatch) // Output from the Fetcher.
```

Comment:
if `fetcher.Run` sent to a `chan` of individual `Entry` could you pass in `jobs` and get rid of the go routine below?

Reply:
Yes, I initially thought to do so. But I can see benefits of having contiguous batches output by the Fetcher, rather than having them mixed in one channel. For example, sending them all together in one AddSequencedLeaves call seems like a good idea to me.

Reply:
Maybe we could do a callback(EntryBatch) instead of a channel, so that to make flattening as cheap as it was before.

Reply:
Done.